pages/index.html

```diff
@@ -44,107 +44,93 @@
         </script>
     </head>
 
-    <body id="search" class="CoveoSearchInterface" data-enable-history="true" style="padding: 1em;">
-        <span class="CoveoAnalytics"></span>
```

Comment:
Why is the Analytics gone?

Reply:
As I said, it's based on the Search page template of the coveo/search-ui.
https://github.com/coveo/search-ui/blob/master/pages/All.html I invite to check it out, you won't find an Analytics.